protocol/avdl/keybase1/config.avdl

```diff
@@ -195,6 +195,7 @@ protocol config {
     boolean isNull;
     union { null, boolean } b;
     union { null, int } i;
+    union { null, double } d;
```

Comment:
Wasn't sure whether it made the most sense to call this field `f` or `d`.  It's a bit weird using the `D` field in the Go code, but it felt right here.

Reply:
id say slight preference for whichever word / actual type (double vs float) is going to be used more frequently. looks like it's float from this PR. but i wouldn't go to the mat over this. 🤷‍♂

Reply:
Changed to `f`.